pyproject.toml

```diff
@@ -15,14 +15,15 @@ classifiers = [
     "Programming Language :: Python :: 3.9",
     "Programming Language :: Python :: 3.10",
     "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
 ]
 dynamic = ["version"]
 readme = "README.md"
 dependencies = [
     "numpy>=1.24,<2.0",
     "cloudpickle>=1.6.0",
-    "substra~=0.53.0",
-    "substratools~=0.21.4",
+    "substra~=0.54.0a1",
+    "substratools~=0.22.0a2",
```

Comment:
To recap:
* `substratools` in its alpha version is needed because we pip install it in the docker template
* `substra` in its alpha version is needed because?

Reply:
My understanding is that we do need the `substratools` release for the python setup in docker. I would like to understand why do we need the `substra` pre-release. If you don't know it's ok, feel free to resolve the comment.